README.md

```diff
@@ -101,249 +142,725 @@ The ```.url()``` method accepts the following parameters.
+The method accepts only jobId.
+
+```python
+result = imagekit.get_bulk_job_status("job_id")
+print("======FINAL RESULT=======")
+print("-------------------------------------")
+print(result)
+print("Raw Response:")
+print(result['response']['_response_metadata']['raw'])
+```
+
+**21. Purge Cache**
+
+Programmatically issue a cache clear request as per
+the [API documentation here](https://docs.imagekit.io/api-reference/media-api/purge-cache).
 Accepts the full URL of the file for which the cache has to be cleared.
+
 ```python
-imagekit.purge_file_cache(full_url)
+result = imagekit.purge_file_cache("full_url")
+print("======FINAL RESULT=======")
+print("-------------------------------------")
+print(result)
+print("Raw Response:")
+print(result['response']['_response_metadata']['raw'])
 ```
-**7. Purge Cache Status**
 
-Get the purge cache request status using the request ID returned when a purge cache request gets submitted as pet the
+**22. Purge Cache Status**
+
+Get the purge cache request status using the request ID returned when a purge cache request gets submitted as per the
 [API documentation here](https://docs.imagekit.io/api-reference/media-api/purge-cache-status)
 
 ```python
-imagekit.get_purge_file_cache_status(cache_request_id)
+result = imagekit.get_purge_file_cache_status("cache_request_id")
+print("======FINAL RESULT=======")
+print("-------------------------------------")
+print(result)
+print("Raw Response:")
+print(result['response']['_response_metadata']['raw'])
+```
+
+**23. Get File Metadata**
+
+Accepts the file ID and fetches the metadata as per
+the [API documentation here](https://docs.imagekit.io/api-reference/metadata-api/get-image-metadata-for-uploaded-media-files)
+
+```python
+result = imagekit.get_file_metadata("file_id")
+print("======FINAL RESULT=======")
+print("-------------------------------------")
+print(result)
+print("Raw Response:")
+print(result['response']['_response_metadata']['raw'])
+```
+
+**24. Get File Metadata from remote url**
+
+Accepts the remote file url and fetches the metadata as per
+the [API documentation here](https://docs.imagekit.io/api-reference/metadata-api/get-image-metadata-from-remote-url)
+
+```python
+result = imagekit.get_remote_file_url_metadata("remote_file_url")
+print("======FINAL RESULT=======")
+print("-------------------------------------")
+print(result)
+print("Raw Response:")
+print(result['response']['_response_metadata']['raw'])
+```
+
+**25. Create CustomMetaDataFields**
+
+Accepts an options specifying the parameters to be used to create cusomMetaDataFields. All parameters specified in
+the [API documentation here](https://docs.imagekit.io/api-reference/custom-metadata-fields-api/create-custom-metadata-field)
+can be passed as it is with the correct values to get the results.
+
+Check for
+the [Allowed Values In The Schema](https://docs.imagekit.io/api-reference/custom-metadata-fields-api/create-custom-metadata-field#allowed-values-in-the-schema-object)
```

Comment:
Please don't use the start case. Simply write `allowed values in the schema` instead of `Allowed Values In The Schema`. This feedback is for all such similar usage. Use sentence case
```
This Sentence Is Called As Start Case. <--- Don't use this
This Sentence is Called as Title Case.
this Sentence Is Called As camelCase.
This sentence is called as sentence case. <-- Use this instead
this_sentence_is_called_as_snake_case.
this-sentence-is-called-as-kebab-case.
```

Reply:
Pending